torch/onnx/symbolic_opset9.py

```diff
@@ -2201,8 +2202,9 @@ def batch_norm(
         return res
 
 
-@symbolic_helper.parse_args("v", "is", "v", "v", "f", "i")
-def layer_norm(g, input, normalized_shape, weight, bias, eps, cudnn_enable):
+def _layer_norm_shared(
+    g, input, normalized_shape, weight, bias, eps, cudnn_enable, return_mean_rstd
```

Comment:
return_mean_rstd: naming:

readability: Avoid abbreviations - they add cognitive load to readers.

> Do not use abbreviations that are ambiguous or unfamiliar to readers outside the project, and do not abbreviate by deleting letters within a word.

https://google.github.io/styleguide/pyguide.html#316-naming
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Reply:
Nice one. This is worthy to be mentioned to most important PRs to Pytorch.  I will replace `_dropout_shared` with `_dropout_returns_masked_input_and_mask` and `_layer_norm_shared` with `_layer_norm_returns_normalized_input_mean_rstd`.

Reply:
Thanks! Do you have thoughts on the parameter name?

Reply:
I prefer to keep them as they are. If we change them, they won't match aten::layer_norm's naming convention. For this kind of problems, my solution would be improving documentation for aten::layer_normin `pytorch/aten/src/ATen/native/native_functions.yaml`. This is a fundamental problem in Pytorch codebase --- no math equation for math functions.

Reply:
I see: I didn't realize rstd was defined in native_functions. Thanks for the clarification!